src/js/DOM.js

```diff
@@ -0,0 +1,95 @@
+import { getPublishedTime, checkDuplicateID } from "./utils.js";
+
+const MODAL = 1;
+
+export const cannotFoundKeyword = () => {
+  return `
+  <div>
+    <img class="js-not-found"
+    src="src/images/status/not_found.png"
+    alt="searchResult not found">
+    <p class="js-not-found font-semibold"> 검색 결과를 찾을 수 없습니다! </p>
+  </div>`;
+};
+
+export const buildResultSection = (dataAry, savedVideos, position) => {
+  let resultDomElement = ``;
+
+  for (let data of dataAry) {
+    resultDomElement += buildVideoArticle(data, savedVideos, position);
+  }
+  return resultDomElement;
```

Comment:
Array의 Map method를 사용해보세요!
```suggestion
  return dataAray.map(data => buildVideoArticle(data, savedVideos, position)).join('');
```

Reply:
머릿속에 기억해둬도 막상 쓸려고 하면 생각 나지 않는 각종 for 관련 메서드들.. 이젠 익숙해질 때도 된 것 같은데 항상 이렇게 빼먹는 것 같습니다 ㅠ